Execution failed for task ':react-native-vision-camera:externalNativeBuildDebug' 🐛

### What were you trying to do?

I can't build for debug

### Reproduceable Code

_No response_

### What happened instead?

I'm getting the folowing errors:
error: redefinition of 'JHashMap'
error: redefinition of 'create'


### Relevant log output

```shell
Build VisionCamera_armeabi-v7a
ninja: Entering directory `C:\Some\Path\Here\ReactNativeProject\node_modules\react-native-vision-camera\android\.cxx\cmake\debug\armeabi-v7a'
In file included from ../../../../src/main/cpp/java-bindings/JHashMap.cpp:5:
../../../../src/main/cpp/java-bindings/JHashMap.h:16:8: error: redefinition of 'JHashMap'
../../../../build/fbjni-0.3.0-headers.jar\fbjni/detail/Iterator.h:170:8: note: previous definition is here
../../../../src/main/cpp/java-bindings/JHashMap.cpp:15:43: error: redefinition of 'create'
../../../../build/fbjni-0.3.0-headers.jar\fbjni/detail/Iterator-inl.h:230:42: note: previous definition is here
In file included from ../../../../src/main/cpp/JSIJNIConversion.cpp:25:
../../../../src/main/cpp/java-bindings/JHashMap.h:16:8: error: redefinition of 'JHashMap'
../../../../build/fbjni-0.3.0-headers.jar\fbjni/detail/Iterator.h:170:8: note: previous definition is here
```


### Device

Xiaomi K20Pro

### VisionCamera Version

2.9.3

### Additional information

- [ ] I am using Expo
- [X] I have read the [Troubleshooting Guide](https://mrousavy.github.io/react-native-vision-camera/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar issues in this repository](https://github.com/mrousavy/react-native-vision-camera/issues) and found none.

--- a/android/src/main/cpp/JSIJNIConversion.cpp
+++ b/android/src/main/cpp/JSIJNIConversion.cpp
@@ -156,10 +156,10 @@ jsi::Value JSIJNIConversion::convertJNIObjectToJSIValue(jsi::Runtime &runtime, c
     auto array = toArrayListFunc(object.get());
     return convertJNIObjectToJSIValue(runtime, array);
 
-  } else if (object->isInstanceOf(jni::JHashMap<jstring, jobject>::javaClassStatic())) {
+  } else if (object->isInstanceOf(JHashMap<jstring, jobject>::javaClassStatic())) {
     // HashMap<K, V>
 
-    auto map = static_ref_cast<jni::JHashMap<jstring, jobject>>(object);
+    auto map = static_ref_cast<JHashMap<jstring, jobject>>(object);
 
     auto result = jsi::Object(runtime);
     for (const auto& entry : *map) {
@@ -173,7 +173,7 @@ jsi::Value JSIJNIConversion::convertJNIObjectToJSIValue(jsi::Runtime &runtime, c
   } else if (object->isInstanceOf(react::ReadableMap::javaClassStatic())) {
     // ReadableMap
 
-    static const auto toHashMapFunc = react::ReadableMap::javaClassLocal()->getMethod<jni::JHashMap<jstring, jobject>()>("toHashMap");
+    static const auto toHashMapFunc = react::ReadableMap::javaClassLocal()->getMethod<JHashMap<jstring, jobject>()>("toHashMap");
 
     // call recursive, this time HashMap<K, V>
     auto hashMap = toHashMapFunc(object.get());

--- a/android/src/main/cpp/java-bindings/JHashMap.cpp
+++ b/android/src/main/cpp/java-bindings/JHashMap.cpp
@@ -7,14 +7,14 @@
 #include <jni.h>
 #include <fbjni/fbjni.h>
 
+namespace vision {
 
-namespace facebook {
-namespace jni {
+using namespace facebook;
+using namespace jni;
 
 template <typename K, typename V>
 local_ref<JHashMap<K, V>> JHashMap<K, V>::create() {
   return JHashMap<K, V>::newInstance();
 }
 
-} // namespace jni
-} // namespace facebook
+} // namespace vision

--- a/android/src/main/cpp/java-bindings/JHashMap.h
+++ b/android/src/main/cpp/java-bindings/JHashMap.h
@@ -7,9 +7,10 @@
 #include <jni.h>
 #include <fbjni/fbjni.h>
 
+namespace vision {
 
-namespace facebook {
-namespace jni {
+using namespace facebook;
+using namespace jni;
 
 // TODO: Remove when fbjni 0.2.3 releases.
 template <typename K = jobject, typename V = jobject>
@@ -19,5 +20,4 @@ struct JHashMap : JavaClass<JHashMap<K, V>, JMap<K, V>> {
   static local_ref<JHashMap<K, V>> create();
 };
 
-} // namespace jni
-} // namespace facebook
+} // namespace vision

